src/Symfony/Component/HttpClient/Tests/HttpClientTestCase.php

```diff
@@ -327,7 +328,12 @@ private static function startVulcain(HttpClientInterface $client)
             'KEY_FILE' => __DIR__.'/Fixtures/tls/server.key',
             'CERT_FILE' => __DIR__.'/Fixtures/tls/server.crt',
         ]);
-        $process->start();
+
+        try {
+            $process->start();
+        } catch (RuntimeException $e) {
+            self::markTestSkipped('vulcain failed: '.$e->getMessage());
+        }
```

Comment:
Unrelated change?

Reply:
No, by using `proc_open()` with an array, starting the process can fail directly since PHP 8.3 if the binary does not exist. This change is needed to restore the desired behavior of that test.